week1/homework/src/index.js

```diff
@@ -1,3 +1,53 @@
-'use strict';
+/* eslint-disable no-unused-vars */
```

Comment:
Why? There aren't any unused variables.

Reply:
I added it so that eslint does not underline the last variable which is badUrl,,, I didn't use it as you see. (But yes I could delete it because I never used it anywhere).